src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1645,7 +1617,7 @@ private function dumpValue($value, bool $interpolate = true): string
     private function dumpLiteralClass(string $class): string
     {
         if (false !== strpos($class, '$')) {
-            return sprintf('${($_ = %s) && false ?: "_"}', $class);
+            return "\${(\$_ = {$class}) ? '_' : '_'}";
```

Comment:
what is this about actually ?

Reply:
the exact same written in a nicer way, but unrelated, now reverted